src/js/api/table.js

```diff
@@ -0,0 +1,90 @@
+export const getNasaData = () => Promise.resolve({ json: () => Promise.resolve(nasaData)});
```

Comment:
1. please use export default syntax
2. consider using default promise api `.then(res => res.json())`

Reply:
It will make sense if we will have more than 1 function here and it seems to me that we won't use any other functions here. But if you mind let me know and I will change it.

Reply:
resolved